templates/repo/actions/runs_list.tmpl

```diff
@@ -28,14 +28,14 @@
 			</div>
 			<div class="flex-item-trailing">
 				{{if .RefLink}}
-					<a class="ui label tw-px-1 tw-mx-0" href="{{.RefLink}}">{{.PrettyRef}}</a>
+					<a class="ui label run-list-ref gt-ellipsis" href="{{.RefLink}}">{{.PrettyRef}}</a>
 				{{else}}
-					<span class="ui label tw-px-1 tw-mx-0">{{.PrettyRef}}</span>
+					<span class="ui label run-list-ref gt-ellipsis">{{.PrettyRef}}</span>
 				{{end}}
```

Comment:
I think the `if` block could be simplified to one line:

```
<a class="ui label run-list-ref gt-ellipsis" href="{{.RefLink}}">{{.PrettyRef}}</a>
```

When the href is empty, the IIRC `a` behaves the similar as `span`. By removing the `if`, it is also easier to test.

Reply:
No it behaves quite differently, it shows a pointer cursor when there is nothing to click, I'm against this change now that I've tested it.